src/components/Navigation/Navigation.vue

```diff
@@ -31,17 +39,20 @@ export default {
     return {
       navContent: categories
     }
+  },
+  computed: {
+    hasData() {
+      return this.$store.state.accountData.id === undefined ? false : true
+    }
```

Comment:
Maybe you can use vuex store getters : https://vuex.vuejs.org/guide/getters.html